src/EnergyPlus/EMSManager.cc

```diff
@@ -1633,13 +1630,16 @@ namespace EMSManager {
         if (allocated(state.dataAirLoop->PriAirSysAvailMgr)) {
             int numAirLoops = isize(state.dataAirLoop->PriAirSysAvailMgr);
             for (int Loop = 1; Loop <= numAirLoops; ++Loop) {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstrict-aliasing"
                 SetupEMSActuator(state,
                                  "AirLoopHVAC",
                                  state.dataAirSystemsData->PrimaryAirSystems(Loop).Name,
                                  "Availability Status",
                                  "[ ]",
                                  state.dataEMSMgr->lDummy2,
                                  (int &)state.dataAirLoop->PriAirSysAvailMgr(Loop).availStatus);
+#pragma GCC diagnostic pop
```

Comment:
This is the only call to SetupEMSActuator that was violating strict-aliasing, so just pragma around it, no need for a template